static/docs/command-reference/remote/add.md

```diff
@@ -282,15 +242,70 @@ $ dvc remote add myremote gs://bucket/path
 
 <details>
 
+### Click for Aliyun OSS
+
+First you need to setup OSS storage on Aliyun Cloud and then use an S3 style URL
+for OSS storage and make the endpoint value configurable. An example is shown
+below:
+
+```dvc
+$ dvc remote add myremote oss://my-bucket/path
+```
+
+To set key id, key secret and endpoint you need to use `dvc remote modify`.
+Example usage is show below. Make sure to use the `--local` option to avoid
+committing your secrets into Git:
+
+```dvc
+$ dvc remote modify myremote --local oss_key_id my-key-id
+$ dvc remote modify myremote --local oss_key_secret my-key-secret
+$ dvc remote modify myremote oss_endpoint endpoint
+```
+
+You can also set environment variables and use them later, to set environment
+variables use following environment variables:
+
+```dvc
+$ export OSS_ACCESS_KEY_ID="my-key-id"
+$ export OSS_ACCESS_KEY_SECRET="my-key-secret"
+$ export OSS_ENDPOINT="endpoint"
+```
+
+#### Test your OSS storage using docker
```

Comment:
it should be part of the contributing guide

Reply:
This section was already in this doc. I just rearranged the order of these H4 sections. Should we open a separate issue?

Reply:
yep, a strange place for this section - let's open an issue and address it later

Reply:
Done. #850